schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -84,6 +84,7 @@
                                     { "$ref": "http://schema.management.azure.com/schemas/2015-03-01-preview/Microsoft.AppService.json#/resourceDefinitions/gateways" },
                                     { "$ref": "http://schema.management.azure.com/schemas/2015-04-01/Microsoft.NotificationHubs.json#/resourceDefinitions/namespaces" },
                                     { "$ref": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/virtualMachines" }
+									{ "$ref": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/virtualMachineScaleSets" }
```

Comment:
Please move this to somewhere between line 40 and line 69. All the resource types here are parent resources, virtualMachineScaleSets does not have child resources other than default ones.
Also please fix the spacing.